nwbinspector/nwbinspector.py

```diff
@@ -200,24 +217,60 @@ def inspect_all_cli(
     n_jobs: int = 1,
     skip_validate: bool = False,
     detailed: bool = False,
+    stream: bool = False,
+    version_id: Optional[str] = None,
 ):
-    """Primary CLI usage of the NWBInspector."""
+    """
+    Run the NWBInspector via the command line.
+
+    path :
+    Relative path to either a local NWBFile, a local folder containing multiple NWBFiles, a link to a dataset on
```

Comment:
the folder could contain just one nwb file

```suggestion
    Relative path to either a local NWBFile, a local folder containing NWBFiles, a link to a dataset on
```